src/rust/ide/lib/view/src/debug_scenes/documentation_generator_view.rs

```diff
@@ -0,0 +1,80 @@
+#![allow(missing_docs)]
+
+use ensogl::traits::*;
```

Comment:
arent traits part of prelude? If they are,  it means that you haven't linted your code somehow. Please check why linter did not catch it - check linter options in `lib.rs` etc and reply here with your findings please.